### PR TITLE
Web 1 updates

### PR DESCRIPTION
**Pull request by GeekTrainer**
Updates to content, flow, clarity, markdown, etc of Web 1

### CODE REVIEW

File: Web_Fundamentals_1/2_HTML/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Part 2. Hypertext Markup Language
 
+Hyptertext Markup Language (HTML) is the lingua franca of the web. HTML is used to structure, or mark up, the information you wish to display to the user. It's the first language all web developers learn, and it's where we're going to start in our exploration.
```
Comment (meaghanlewis) [inline Web_Fundamentals_1/2_HTML/README.md]: I like the chapter introductions you added!

File: Web_Fundamentals_1/3_CSS_CSS3/display_property.md [hunk 2]
```diff
@@ -1,73 +1,92 @@
-# The Display Property - Block, Inline-Block, Inline
+# The Display Property
 
-Each HTML element by default is either **inline** or **block**.
+Not all tags are created equal, and as a result not all tags are displayed the same way. Understanding how a tag will display by default allows you to better control page layout and predict how items will be displayed. Each HTML element is displayed as either `block` or `inline`.
 
-**Inline** items only take up as much space as needed.
+## inline
```
Comment (meaghanlewis) [inline Web_Fundamentals_1/3_CSS_CSS3/display_property.md]:
```suggestion
## Inline
```

File: Web_Fundamentals_1/3_CSS_CSS3/display_property.md [hunk 3]
```diff
@@ -1,73 +1,92 @@
...
+This will display as:
 
-![block](https://s3.amazonaws.com/General_V88/boomyeah2015/codingdojo/curriculum/content/chapter/Block.png)
+[Microsoft](https://www.microsoft.com) created [Visual Studio Code](https://code.visualstudio.com). Isn't that **cool?!**
 
-Block elements: ```<body>, <div>, <form>, <h1>-<h6>, <p>, <table>, <ul>, <ol>, <li>```
+Some examples of inline elements are `<a>`, `<span>`, `<input>`, `<label>`, `<select>`, and `<textarea>`.
 
-**It's invalid to put block elements inside inline elements.**
+## block
```
Comment (meaghanlewis) [inline Web_Fundamentals_1/3_CSS_CSS3/display_property.md]:
```suggestion
## Block
```

File: Web_Fundamentals_1/3_CSS_CSS3/display_property.md (hunk 2 above)
Comment (meaghanlewis) [reply Web_Fundamentals_1/3_CSS_CSS3/display_property.md]: Just a suggestion to capitalize this header, although you might have done this intentionally.

File: Web_Fundamentals_2/1_HTML/common_body_tags.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ Any chunk of text is a paragraph and, therefore, needs to be encapsulated in par
 
 Here's an example of how to use ```<p>``` tags:
 
-```html
+``` html
```
Comment (meaghanlewis) [inline Web_Fundamentals_2/1_HTML/common_body_tags.md]: It looks like the styling on this page is off.

File: Web_Fundamentals_1/3_CSS_CSS3/display_property.md (hunk 2 above)
Comment (GeekTrainer) [reply Web_Fundamentals_1/3_CSS_CSS3/display_property.md]: It was left lower case to match the casing of the option

File: Web_Fundamentals_2/1_HTML/common_body_tags.md (hunk 4 above)
Comment (GeekTrainer) [reply Web_Fundamentals_2/1_HTML/common_body_tags.md]: I double checked and it seems clean 🤷🏻‍♀️

**Pull request merged**